app/queue.py

```diff
@@ -22,8 +24,11 @@ class Buffer(Enum):
     def inbox_name(self, suffix=None):
         return f"{self.value}:{suffix}" if suffix else self.value
 
+    def inflight_prefix(self, suffix: str = None) -> str:
+        return f"{Buffer.IN_FLIGHT.value}:{str(suffix)}" if suffix else f"{Buffer.IN_FLIGHT.value}"
+
```

Comment:
factored this out to use it again in the `expire_inflights()` code

Reply:
IMHO this needs its own space